pkgs/by-name/we/webrtc-audio-processing_2/package.nix

```diff
@@ -0,0 +1,76 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  fetchurl,
+  darwin,
+  abseil-cpp,
+  meson,
+  ninja,
+  pkg-config,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "webrtc-audio-processing";
+  version = "2.1";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "pulseaudio";
+    repo = "webrtc-audio-processing";
+    rev = "v${version}";
+    hash = "sha256-YR4ELukJgHMbfe80H+r8OiaZUCAqefGXmVOaTVVgOqA=";
+  };
+
+  outputs = [
+    "out"
+    "dev"
+  ];
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+  ];
+
+  propagatedBuildInputs = [
+    abseil-cpp
+  ];
+
+  env = lib.optionalAttrs stdenv.hostPlatform.isx86_32 {
+    # https://gitlab.freedesktop.org/pulseaudio/webrtc-audio-processing/-/issues/5
+    NIX_CFLAGS_COMPILE = "-msse2";
+  };
+
+  mesonFlags = lib.lists.optional (!stdenv.hostPlatform.isAarch64) "-Dneon=disabled";
+
+  meta = with lib; {
+    homepage = "https://www.freedesktop.org/software/pulseaudio/webrtc-audio-processing";
+    description = "More Linux packaging friendly copy of the AudioProcessing module from the WebRTC project";
+    license = licenses.bsd3;
+    platforms =
+      intersectLists
+        # https://gitlab.freedesktop.org/pulseaudio/webrtc-audio-processing/-/blob/master/meson.build
+        (
+          platforms.darwin
+          ++ platforms.linux
+          ++ platforms.windows
+          ++ platforms.freebsd
+          ++ platforms.netbsd
+          ++ platforms.openbsd
+        )
+        # https://gitlab.freedesktop.org/pulseaudio/webrtc-audio-processing/-/blob/master/webrtc/rtc_base/system/arch.h
+        (
+          platforms.arm
+          ++ platforms.aarch64
+          ++ platforms.mips
+          ++ platforms.power
+          ++ platforms.riscv
+          ++ platforms.x86
+          ++ platforms.loongarch64
+        );
+    # BE platforms are unsupported
+    # https://gitlab.freedesktop.org/pulseaudio/webrtc-audio-processing/-/issues/31
+    badPlatforms = platforms.bigEndian;
```

Comment:
Would you be willing to add yourself as a maintainer? It would be much appreciated!